lib/elixir/lib/code/formatter.ex

```diff
@@ -261,6 +257,19 @@ defmodule Code.Formatter do
     }
   end
 
+  @doc """
+  Converts the quoted expression into an algebra document
+  """
+  def quoted_expression_to_algebra(quoted, opts \\ []) do
```

Comment:
Suggestion: call this `to_algebra` and move part of the logic in the current `to_algebra` to `Code.format_string!`. And make sure that `Code.format_string!` call this function instead.

Reply:
This would make `Code.format_string!` use the normalizer, no? If so we would get the tests suggested in this comment: https://github.com/elixir-lang/elixir/pull/10988#discussion_r636748972 for free, right?

Reply:
Done!